browser/net/url_context.cc

```diff
@@ -124,6 +125,11 @@ void BraveRequestInfo::FillCTX(const network::ResourceRequest& request,
       prefs->GetInteger(kIPFSResolveMethod)) ==
           ipfs::IPFSResolveMethodTypes::IPFS_LOCAL;
 #endif
+
+  if (old_ctx) {
```

Comment:
I would add a TODO here saying that this is needed only due to a hacky code in proxying factory.